src/dataset-builder/ConceptSelector.ts

```diff
@@ -87,13 +87,14 @@ export const ConceptSelector = (props: ConceptSelectorProps) => {
             },
           },
           { name: 'Concept ID', width: 195, render: _.get('id') },
+          { name: 'Code', width: 195, render: _.get('code') },
           {
             name: 'Roll-up count',
             width: 205,
             render: (row) => formatCount(row.count),
           },
         ],
-        root: { id: findRoot(parents), name: 'root', count: 0, hasChildren: true },
+        root: { id: findRoot(parents), name: 'root', code: '0', count: 0, hasChildren: true },
```

Comment:
'0' not 0 because concept code is a string right?

Reply:
yep!